src/utils/exportAsImage.ts

```diff
@@ -22,7 +23,7 @@ export const exportStats = async (
           const url = URL.createObjectURL(blob);
           const a = document.createElement("a");
           a.href = url;
-          a.download = "stats.png";
+          a.download = `${filename}.png`;
```

Comment:
`filename` is optional in the function here, we should either make sure that it mandatory when going with `option: "download"` or give it a default